src/main/java/org/sonarlint/intellij/http/ApacheHttpResponse.kt

```diff
@@ -32,7 +32,7 @@ internal class ApacheHttpResponse(
 
   override fun bodyAsString(): String = response.bodyText
 
-  override fun bodyAsStream() = ByteArrayInputStream(response.bodyBytes)
+  override fun bodyAsStream() = ByteArrayInputStream(if(response.bodyBytes != null) response.bodyBytes else ByteArray(0))
```

Comment:
I _think_ the getBodyBytes() method does some computation behind the scenes so maybe we should call it only once ?